APMrover2/mode_guided.cpp

```diff
@@ -145,6 +145,33 @@ bool ModeGuided::reached_destination() const
     return true;
 }
 
+// get desired location
+bool ModeGuided::get_desired_location(Location& destination)
```

Comment:
Is it possible to `const` these methods?

Reply:
it is apparently, changed to be const!